### ISSUE TITLE
Web UI throws unexplainable error

### ISSUE THREAD

**Issue by ghost** [ElektraInitiative/libelektra#2408]
I have mounted a specification as follows:
```
sudo kdb mount ..../server/specification/LCDd-spec.ini 'spec/sw/lcdproc/lcdd/#0/current' ni
sudo kdb spec-mount '/sw/lcdproc/lcdd/#0/current
```
I receive the following error message multiple times:
![error 1](https://user-images.githubusercontent.com/12346621/52594607-3ac6a580-2e4c-11e9-85fe-259a71693a2a.png)
Furthermore I have an empty space on the gui:
![what](https://user-images.githubusercontent.com/12346621/52594632-44e8a400-2e4c-11e9-9a46-f7b4a536cf27.jpg)
The namespace is actually a weird copy of the spec namespace:
![wut2](https://user-images.githubusercontent.com/12346621/52594698-6fd2f800-2e4c-11e9-92ba-6f6d54de7158.png)

#2332

**Comment by markus2330** [comment]
Thank you for reporting!

I am afraid cascading keys are not yet supported in the Web-UI. #2409 and #2410 are because of the same reason.

@omnidan Do you maybe want to add some support? Do you have time? Can you point us to which parts would need changes?

**Comment by markus2330** [comment]
@omnidan Can you point us to which parts would need changes?

**Comment by omnidan** [comment]
@markus2330 I think this could be added in kdb.js which is used by the web ui to interface with the kdb.

**Comment by markus2330** [comment]
Thank you for the quick reply!

Yes, it should be enough to change `kdb <ns>/key` to `kdb -N <ns> /key`. And then of course the ui needs to be fixed so that the cascading keys are not shown (see screenshot above). (Actually this would be automatically fixed once we fix  #2456.)
